tests/updater/controller/v1/test_routes.py

```diff
@@ -83,6 +83,47 @@ async def test_rollout_and_download_info(async_client, test_data):
     )
     assert data["deployment"]["chunks"][0]["artifacts"][0]["size"] == firmware.size
 
+    # confirm installation start (in reality: several of simular posts)
+    response = await async_client.post(
+        f"/DEFAULT/controller/v1/{device.uuid}/deploymentBase/{firmware.id}/feedback",
+        json={
+            "id": firmware.id,
+            "status": {
+                "result": {"finished": "none"},
+                "execution": "proceeding",
+                "details": ["Installing Update Chunk Artifacts."],
+            },
+        },
+    )
+    assert response.status_code == 200
+    await device.refresh_from_db()
+    assert device.last_state == "running"
+
+    # HEAD /api/download/1 HTTP/1.1 (reason not clear)
```

Comment:
There is a good chance this is to check the size of the firmware.  I think we can implement this for local files, can do some research into what a file header should send.

Reply:
You were spot on: looks like swupdate queries the size of the firmware. Implemented endpoint.